src/systemdhook.c

```diff
@@ -677,8 +682,23 @@ int main(int argc, char *argv[])
 	}
 	char *id = YAJL_GET_STRING(v_id);
 
+	const char *bundle_path[] = { "bundlePath", (const char *)0 };
```

Comment:
I think that a comment at the beginning here explaining how this works w.r.t. arguments and configs when used with OCI and docker would help.